<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralSettings</class>
 <widget class="QWidget" name="GeneralSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="docGroup">
     <property name="title">
      <string>文件</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="docPathTitle">
        <property name="text">
         <string>默认保存位置</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="docPath"/>
        </item>
        <item>
         <widget class="QPushButton" name="docPathBrowse">
          <property name="text">
           <string>浏览...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="docStatusTitle">
        <property name="text">
         <string>状态</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="docStatus"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="dbGroup">
     <property name="title">
      <string>数据库</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="4" column="0">
       <widget class="QLabel" name="dbTypeTitle">
        <property name="text">
         <string>数据库类型</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="dbType">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="dbNameTitle">
        <property name="text">
         <string>数据库名</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="dbName"/>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="dbStatusTitle">
        <property name="text">
         <string>状态</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLabel" name="dbStatus">
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="dbHost"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="dbHostTitle">
        <property name="text">
         <string>主机</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="dbUserTitle">
        <property name="text">
         <string>用户名</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="dbPasswordTitle">
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="dbUser"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="dbPassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
